### PR TITLE
DS Ethiop 1-5

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Small chances

### CODE REVIEW

**Comment by DenisNosnitsin1970** [comment]
Jetzt?

**Comment by thea-m** [comment]
Ja!

File: DayrSuryan/DSEthiop3.xml [hunk 1]
```diff
@@ -759,9 +758,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <layoutDesc>
                            <layout columns="1" writtenLines="21 24">
                                 <locus from="101v" to="102v"/>
-                                <note corresp="#textarea2">Data on text area dimensions taken from <locus target="#101v"/>.</note>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop3.xml:762]: the corresp and the xml:id must correspond (which indeed they didn't...), I would propose textarea2. There are no margin measurements, so #margins can be deleted.

File: DayrSuryan/DSEthiop4.xml [hunk 2]
```diff
@@ -461,11 +461,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <locus target="#172v"/>
                         <desc>A European printed devotional image is glued on the folium,
                           depicting St Mary with Child, with a quote
-                           in Latin (Song 2;16) <foreign xml:lang="la">“Dilectus meus et ego illi qui pascitur inter lilia (Cant.
-                                   2-16).”</foreign> Legends in Ethiopic, in crude hand, are added above the image, <foreign xml:lang="gez">ምስለ፡ ፍቁር፡ ወልዳ፡ ወላዲተ፡ አምላክ፡ </foreign>
+                           in Latin (Song 2;16) <foreign xml:lang="la">“Dilectus meus mihi et ego illi qui pascitur inter lilia (Cant.
+                                   2-16).”</foreign> Legends in Ethiopic, in crude hand, are added above the image,
```
Comment (thea-m) [inline DayrSuryan/DSEthiop4.xml:465]: in a crude hand?

**Review by thea-m — CHANGES_REQUESTED** [review]
small remarks

File: DayrSuryan/DSEthiop2.xml [hunk 3]
```diff
@@ -686,7 +686,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                               <desc type="DonationNote">Donation note stating that this manuscript belongs to
                                  <persName ref="PRS11939TaklaH"/>, who donated it to <placeName ref="INS0479DabraM"/>. In the same hand
                               as <ref target="#a9">Addition 9</ref>.</desc>
-                              <q xml:lang="gez">ዛቲ፡ መጽሐፍ፡ ዘ<persName ref="PRS11939TaklaH"><roleName type="title">አቡነ</roleName>ተክለ፡ ሃማኖት፡</persName>
+                              <q xml:lang="gez">ዛቲ፡ መጽሐፍ፡ ዘ<persName ref="PRS11939TaklaH"><roleName type="title">
+                                <supplied reason="omitted" resp="DN">፡</supplied> አቡነ</roleName>ተክለ፡ <sic resp="DN">ሃማኖት፡</sic></persName>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop2.xml:690]: sorry, false place

File: DayrSuryan/DSEthiop3.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop3.xml:762]: ok

File: DayrSuryan/DSEthiop4.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop4.xml:465]: ok

**Review by thea-m — APPROVED** [review]

**Pull request merged**